src/components/toolbar/search-component.css

```diff
@@ -48,16 +50,24 @@
   cursor: pointer;
 }
 
-#search .search-result {
-  color: white;
-}
-
-#search .search-result:hover {
+#search .search-result-select {
+  display: flex;
+  flex-direction: column;
+  padding: 8px 8px;
+  cursor: pointer;
   color: purple;
   background: rgba(0, 0, 0, 0.0625);
 }
 
-.theme-dark #search .search-result:hover {
+#search .search-result {
+  color: white;
+}
+
+.theme-dark #search .search-result-select {
+  display: flex;
```

Comment:
When you scope the style under the `.theme-dark` selector, you just need to specify the css rules that need to be different under this scope. In this case, the only thing that needs to change is the background color. The rules defined in `#search .search-result-select` still apply so no need to duplicate them in this dark theme variant scope.